docs/gene_sets.md

```diff
@@ -0,0 +1,103 @@
+## Overview
+
+Users may view gene sets along with their dataset and access gene sets created in pip installed cellxgene by following the instructions below.
```

Comment:
I suggest defining a `geneset` before using it.   Ie, "In cellxgene, a geneset is ...`